### PR TITLE
Secondbranch

### PR DESCRIPTION
**Pull request by Lejla1502**
Deleted extra headers and edited existing one inside of contributersProfile.html

### CODE REVIEW

File: contributersProfile.html [hunk 1]
```diff
@@ -445,13 +445,18 @@ <h2 class="animated bounce">Contributors</h2>
...
+            <div class="dropDownContent"></div>
+          </li>
         </ul>
       </div>
       <div class="container-fluid footer-container">
         <div class="a2a_kit a2a_kit_size_32 a2a_default_style" style="line-height: 32px;">
           <h5>
-            If you enjoyed the course like I did, share it so more people can
-            enjoy it!
+            If you enjoyed or liked the work of our contributors, you can go to their profile and leave
```
Comment (thuongtruong1009) [inline contributersProfile.html:458]: Because "enjoyed" is similar to "liked", so you should simplify do it.

File: resources/style/particles.css [hunk 2]
```diff
@@ -2,58 +2,190 @@
...
+nav,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
```
Comment (thuongtruong1009) [inline resources/style/particles.css:76]: This CSS style should be border: none and outline: none

File: resources/style/particles.css [hunk 3]
```diff
@@ -62,34 +194,33 @@ canvas{
...
-
 /* ---- particles.js container ---- */
 
-#particles-js{
+#particles-js {
   width: 100%;
   height: 100%;
   /* background-color: #b61924; */
-  background-image: url('');
+  background-image: url("");
```
Comment (thuongtruong1009) [inline resources/style/particles.css:223]: remove this line because they empty

File: resources/style/style.css [hunk 4]
```diff
@@ -205,9 +205,9 @@ nav.navbar button.navbar-toggler {
 
 .header-content h1 {
   width: 100%;
-  margin-top: 80px;
+
   text-align: center;
-  padding: 0;
+  padding-bottom: 10;
```
Comment (thuongtruong1009) [inline resources/style/style.css:210]: fix to: padding-bottom: 10px;

**Review by thuongtruong1009 — APPROVED** [review]
Thank you for your PR, however, you need to modify a few things to make the product more complete. Thank you!

**Review by thuongtruong1009 — APPROVED** [review]
Very good. Keep contributing.
Thank you very much!

**Comment by Lejla1502** [comment]
Thank you so much for dedicating your time to look through my code. I appreciate it very much.

**Pull request merged**